modules/service-esm.sh

```diff
@@ -9,8 +9,8 @@ esm_enable() {
     local token="$1"
 
     check_token "$ESM_REPO_URL" "$token"
```

Comment:
It's not a problem introduced in this branch, but it will show up only once this lands. We should check if esm is already enabled before enabling it again, or else we will pile up machine lines in auth.conf, not to mention running apt update multiple times needlessly.

Reply:
Right, we should check if the service is enabled as first thing for each service. I'd defer this to a separate branch, since it's an issue already and I plan to unify the enable/disable logic.
I opened #99